test/fuzzing/main.cpp

```diff
@@ -0,0 +1,23 @@
+// if an int main() is needed.
+#if FMT_FUZZ_IMPLEMENT_MAIN
+#  include <cassert>
+#  include <fstream>
+#  include <sstream>
```

Comment:
Unused.

Reply:
I removed the macro, and med the inclusion of main.cpp conditional in the cmake file as per your suggestion. It is used in case of building a reproducer or using afl. You can see it in action if you have the patience to wait for build.sh finishing the afl build :-)